components/brave_today/common/brave_news.mojom

```diff
@@ -102,6 +117,9 @@ struct DisplayAd {
 interface BraveNewsController {
   GetFeed() => (Feed feed);
   GetPublishers() => (map<string, Publisher> publishers);
+  SubscribeToNewDirectFeed(url.mojom.Url feed_url)
```

Comment:
nit: How about adding some comments something like this gives latest publishers list when new `feed_url` is valid?

Reply:
added in https://github.com/brave/brave-core/compare/f4f1b4d6eb6fa0acbfb1c93897d26ac6f265e948..7aec98389bea9a3bb41d706631a13d9f3a7fe1fb